Week1/solution/solution6.py

```diff
@@ -0,0 +1,8 @@
+"""Write a python script to enter any string and print only part of string.
+Take input of start character and end character from user."""
+name=input('Enter string :')
+sc=input('Enter start character :')
+ec=input('Enter end character :')

```

Comment:
if multiple character are present like 'aaruhi' use input for 2nd 'a 'and and .index() method gives first 'a' index . i think in this case program not give expected output.